src/App.js

```diff
@@ -10,7 +10,7 @@ const App = () => (
       <Header>
         <Routes>
           <Route path="/" element={<Home />} />
-          <Route path="/details" element={<Details />} />
+          <Route path="/:id" element={<Details />} />
```

Comment:
- [ ] please ensure that you deploy the application using Heroku, Netlify or GitHub Pages.
- [ ] Kindly add a link to the deployed project in the README.